SECURITY.md

```
@@ -0,0 +1,12 @@
+# Security Policy
+
+## Reporting a Vulnerability
+
+If you discover a security vulnerability, **please report it privately** using GitHub’s Private Vulnerability Reporting feature:
+
+https://github.com/kauntdewn1/interbox-v2/security/advisories/new
```

markdownlint-cli2 (0.18.1)

Bare URL used

(MD034, no-bare-urls)